components/camera/server.go

```diff
@@ -90,6 +92,88 @@ func (s *serviceServer) GetImage(
 	return &resp, nil
 }
 
+// GetImages returns a list of images and metadata from a camera of the underlying robot.
+func (s *serviceServer) GetImages(
+	ctx context.Context,
+	req *pb.GetImagesRequest,
+) (*pb.GetImagesResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "camera::server::GetImages")
+	defer span.End()
+	cam, err := s.coll.Resource(req.Name)
+	if err != nil {
+		return nil, err
```

Comment:
[nit] Could we wrap these error messages with additional text so we have context if this throws an error? Here and throughout.

Reply:
done